BatteryStateManagement_proj.c

```diff
@@ -0,0 +1,209 @@
+/*****************************************************************************/
+/***    \file        BatteryStateManagement_proj.c
+***     \author      Naveen Balachandar (RBEI/ECK3) 
+***
+***     \brief       File to monitor the battery condition based required properties
+/*****************************************************************************/
+/*------ standard includes -------*/
+
+#include <stdio.h>
+#include <assert.h>
+
+/*Macros*/
+# define GERMAN_LANG_SLECTD 1
+# define ENGLANG_LANG_SLECTD 0
+ 
+/*------ Global variables -------*/
+unsigned int langSelected_uint = GERMAN_LANG_SLECTD;/*language selecter*/
+const char batPar[6][12] = {"temp","soc","chargerate","temp","Ladezustand","Ladestrom"}; /*Battery par printed for ref lang: German and english*/
```

Comment:
Any test to ensure that future maintainers don't miss a translation when they add something?